packages/wallet/ui/src/components/ConnectionSettingsDialog.jsx

```diff
@@ -127,7 +127,7 @@ const ConnectionSettingsDialog = ({
                 break;
               case 'localhost':
                 setConfig({
-                  href: `http://localhost:8000/wallet/network-config`,
+                  href: `http://localhost:3000/wallet/network-config`,
```

Comment:
8000 was never used anymore so just change this to 3000

Reply:
Ok great. We should update all the docs then too. 8000 is most of what I found when searching

Reply:
8000 was used for ag-solo I think which is probably why